src/main/scala/com/advancedtelematic/director/db/EcuUpdateAssignmentMigration.scala

```diff
@@ -0,0 +1,61 @@
+package com.advancedtelematic.director.db
+
+import akka.actor.ActorSystem
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Sink, Source}
+import akka.Done
+import com.advancedtelematic.libats.data.DataType.Namespace
+import com.advancedtelematic.libats.slick.db.SlickValidatedGeneric.validatedStringMapper
+import org.slf4j.LoggerFactory
+import slick.jdbc.GetResult
+import slick.jdbc.MySQLProfile.api._
+import com.advancedtelematic.libats.slick.db.SlickAnyVal._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class EcuUpdateAssignmentMigration(implicit
+  val db: Database,
+  val mat: Materializer,
+  val system: ActorSystem,
+  val ec: ExecutionContext)
+  extends DeviceUpdateAssignmentRepositorySupport
+  with EcuUpdateAssignmentRepositorySupport {
+
+  private val _log = LoggerFactory.getLogger(this.getClass)
+
+  implicit val getResult: GetResult[Namespace] = pr => {
+    implicitly[ColumnType[Namespace]].getValue(pr.rs, 1)
+  }
+
+  def run: Future[Done] = {
+    val source = db.stream(Schema.deviceTargets.result)
```

Comment:
```sql
select dt.*, e.namespace from device_update_targets dt inner join ecus e on e.device = dt.device
```
But anyway not really important

Reply:
How to get a `DeviceUpdateTarget` out of that query?

Reply:
You'd need to write a `GetResult[(DeviceUpdateTarget, Namespace)]`